src/components/sendbird/ChannelMembersPreview.tsx

```diff
@@ -24,39 +24,42 @@ const ChannelMembersPreview = React.memo(
     }
 
     return (
-      <Row
-        style={[
-          styles.container,
-          containerStyle,
-          { paddingStart: (-1 * size) / 2 },
-        ]}
-      >
+      <Row style={[styles.container, containerStyle]}>
```

Comment:
can you post the screenshot of how this looks? Please verify that this Component looks good for "BOTH" chatchard and channel settings screen

Reply:
Design
![스크린샷 2023-06-28 오후 2 54 59](https://github.com/palmapp-xyz/palm-mobile/assets/35752384/b37e4ab1-e4c1-4a84-89f9-d7b0ec7b07c6)

App
![스크린샷 2023-06-28 오후 2 57 44](https://github.com/palmapp-xyz/palm-mobile/assets/35752384/597c5a6f-a1fa-461d-9525-29386fc68225)